docs/deployment/on_kubernetes.md

```diff
@@ -5,3 +5,97 @@
 </div>
 
 # Running Kyuubi on Kubernetes
+
+## Requirements
+
+When you want to deploy Kyuubi’s Spark SQL engines on Kubernetes(use K8S as the name below), you’d better have cognition
+upon the following things.
+
+* Knowing the basics about [Running Spark on Kubernetes](http://spark.apache.org/docs/latest/running-on-kubernetes.html)
+* A binary distribution of Spark which is built with K8S
+* An active K8S cluster
+* An active Apache Hadoop HDFS cluster
+* Setup Hadoop client configurations at the machine which Kyuubi server locates
+
+## Configurations
+
+### Environment
+
+Either `HADOOP_CONF_DIR` is configured and points to the Hadoop client configurations directory,
+usually, `$HADOOP_HOME/etc/hadoop`.
+
+If the `HADOOP_CONF_DIR` points the K8S and HDFS cluster correctly, you should be able to run the `SparkPi` example on
```

Comment:
A Pi job have nothing to do with hdfs with k8s

Reply:
so, wo should provide another example, or just remove this `HADOOP_CONF_DIR`